crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -133,6 +133,19 @@ async fn symbolicate_js_frame(
 
     let (line, col) = match (raw_frame.lineno, raw_frame.colno) {
         (Some(line), Some(col)) if line > 0 && col > 0 => (line, col),
+        // Without a column we can still at least apply source context
+        (Some(line), _) if line > 0 => {
+            let filename = frame.raw.filename.as_ref();
+            let file_key = filename.and_then(|filename| module.source_file_key(filename));
+
+            let source_file = match file_key {
+                Some(key) => lookup.get_source_file(key).await,
+                None => &Err(CacheError::NotFound),
+            };
+
+            apply_source_context_from_artifact(&mut frame.raw, source_file);
+            return Ok(frame);
```

Comment:
I wonder if we should still support reporting an error here. Wdyt @Swatinem - https://github.com/getsentry/sentry/blob/master/src/sentry/lang/javascript/processor.py#L1483-L1486

Reply:
Yeah that's not a bad idea.

Reply:
I’m not sure this makes sense. What source context will you apply?
The answer would be the minified source context that you have already applied previously.
Because without applying the sourcemap, the `filename` will not exist. Or even if it does, it would be the same as the `abs_path` of the minified file.